controller/main.py

```diff
@@ -342,9 +342,14 @@ def job_to_job_definition(job, task_id):
     if job.requires_db and job.repo_url in config.REPOS_WITH_EHRQL_EVENT_LEVEL_ACCESS:
         env["EHRQL_ENABLE_EVENT_LEVEL_QUERIES"] = "True"
 
+    original_job_request = get_saved_job_request(job)
```

Comment:
I reckon this might be worth an inline comment for future travellers (e.g. "Use the saved job request as at the time of writing `branch` is not part of the Job model, RAP API phase 2 is in progress, and we may ultimately switch ORM.")